README.md

```diff
@@ -9,6 +9,7 @@ Artifacts in support of ODH Edge use cases that integration with Red Hat Advance
 | Red Hat Advanced Cluster Management  | 2.8     |
 | OpenShift Pipelines                  | 1.11.x  |
 | Quay Registry                        | 2.8     |
+| Gitea                                |         |
```

Comment:
We're using `1.20.2`, you could mention that here if you want.

![image](https://github.com/opendatahub-io/ai-edge/assets/99439005/e1d944a0-ee0b-438d-be24-40bcd258040b)

Reply:
Thx. Updated